src/mmw/apps/modeling/tasks.py

```diff
@@ -94,10 +98,12 @@ def get_histogram_job_results(self, incoming):
     """
     pixel_width = incoming['pixel_width']
     histogram = histogram_finish(incoming['sjs_job_id'], self.retry)
+    aoi = incoming['aoi']
 
     return {
         'pixel_width': pixel_width,
-        'histogram': histogram
+        'histogram': histogram,
+        'aoi': aoi
```

Comment:
Instead of thread the area of interest through `start_histogram_job` and `get_histogram_job_results`, just to have them available in `histogram_to_survey`, consider giving it directly to that function:

```diff
diff --git a/src/mmw/apps/modeling/tasks.py b/src/mmw/apps/modeling/tasks.py
index 0ac6b7e..beb3c89 100644
--- a/src/mmw/apps/modeling/tasks.py
+++ b/src/mmw/apps/modeling/tasks.py
@@ -108,7 +108,7 @@ def get_histogram_job_results(self, incoming):
 
 
 @shared_task
-def histogram_to_survey(incoming):
+def histogram_to_survey(incoming, aoi):
     """
     Converts the histogram results (aka analyze results)
     to a survey of land use, which are rendered in the UI.
@@ -117,7 +117,7 @@ def histogram_to_survey(incoming):
     data = incoming['histogram'][0]
     results = data_to_survey(data)
     convert_result_areas(pixel_width, results)
-    results.append(animal_population(incoming['aoi']))
+    results.append(animal_population(aoi))
 
     return results
 
diff --git a/src/mmw/apps/modeling/views.py b/src/mmw/apps/modeling/views.py
index 36bd598..5f1390e 100644
--- a/src/mmw/apps/modeling/views.py
+++ b/src/mmw/apps/modeling/views.py
@@ -364,7 +364,7 @@ def _initiate_analyze_job_chain(area_of_interest, job_id, testing=False):
                  .set(exchange=exchange, routing_key=routing_key),
                  tasks.get_histogram_job_results.s()
                  .set(exchange=exchange, routing_key=routing_key),
-                 tasks.histogram_to_survey.s()
+                 tasks.histogram_to_survey.s(area_of_interest)
                  .set(exchange=exchange, routing_key=choose_worker()),
                  save_job_result.s(job_id, area_of_interest)
                  .set(exchange=exchange, routing_key=choose_worker())) \
```